### PR TITLE
update telemetry event endpoint

### PR DESCRIPTION
**Pull request by creative-dev-lab**
### Context
telemetry event endpoint update.

### What's updated/implemented
- gradle update
- implemented `DateUtils` to get ISO format date string and used it to add an additional param called `created_at`
- updated telemetry event endpoint

### Release type
- [x] Patch
- [ ] Minor
- [ ] Major

### Peer review
- [x] The feature/changes have been peer reviewed
- [ ] Automated test coverage and correctness have been peer reviewed
- [ ] The test plan has been peer reviewed

### CODE REVIEW

File: AirRobeWidget/src/main/kotlin/com/airrobe/widgetsdk/airrobewidget/service/api_controllers/AirRobeTelemetryEventController.kt [hunk 1]
```diff
@@ -48,11 +50,11 @@ internal class AirRobeTelemetryEventController {
...
             param.put("properties", properties)
 
-            val response = AirRobeApiService.requestPOST(
+            val response = AirRobeApiService.requestPUT(
                 if (config.mode == Mode.PRODUCTION)
-                    AirRobeConstants.AIRROBE_CONNECTOR_PRODUCTION + "/telemetry_events"
+                    TelemetryEventHost.Production.raw + "/v1"
                 else
-                    AirRobeConstants.AIRROBE_CONNECTOR_SANDBOX + "/telemetry_events",
+                    TelemetryEventHost.Sandbox.raw + "/va",
```
Comment (johnAirRobe) [inline AirRobeWidget/src/main/kotlin/com/airrobe/widgetsdk/airrobewidget/service/api_controllers/AirRobeTelemetryEventController.kt:57]: Typo here, mean to be `v1`

**Review by johnAirRobe — CHANGES_REQUESTED** [review]

File: AirRobeWidget/src/main/kotlin/com/airrobe/widgetsdk/airrobewidget/service/api_controllers/AirRobeTelemetryEventController.kt (hunk 1 above)
Comment (creative-dev-lab) [reply AirRobeWidget/src/main/kotlin/com/airrobe/widgetsdk/airrobewidget/service/api_controllers/AirRobeTelemetryEventController.kt:57]: oh yea! 😆

**Comment by johnAirRobe** [comment]
Were you going to add the sdk version to this PR? Or create a new PR for it?

**Comment by creative-dev-lab** [comment]
> Were you going to add the sdk version to this PR? Or create a new PR for it?

Oh, Right. almost forgot this.
I already merged the iOS one, so guess in the next PR if it's okay.

**Review by johnAirRobe — APPROVED** [review]

**Pull request merged**